README.md

```diff
@@ -1302,7 +1302,7 @@ Below is the output for `/path/to/vmselect -help`:
   -blockcache.missesBeforeCaching int
      The number of cache misses before putting the block into cache. Higher values may reduce indexdb/dataBlocks cache size at the cost of higher CPU and disk read usage (default 2)
   -cacheDataPath string
-     Path to directory for cache files. By default, the cache is not persisted.
+     Path to directory for cache files. By default, the cache is not persisted. Please note that vmselect will create a `/tmp/searchResults` folder under cacheDataPath, which contains data from vmstorage nodes for search query. Ensure that `cacheDataPath/tmp/searchResults` will not be removed. Refer to issue #5770 for more details.
```

Comment:
> Refer to issue #5770 for more details

What do you think about adding more details to the documentation and adding a link to the documentation here?
That should make it easier to find relevant content.

Reply:
Agree, but I didn't find a place to make a note of it on the [cluster-victoriametrics](https://docs.victoriametrics.com/cluster-victoriametrics/) page.

My second choice is [troubleshooting](https://docs.victoriametrics.com/troubleshooting/), but it seems like most of the issues addressed here are significant and common. I don't think the issue in this PR carries the same weight. WDYT

Reply:
I agree with @zekker6

This flag should be update with just `By default, the cache is not persisted.` to `By default, the cache is stored at /tmp/searchResults.`

All other information seems misplaced because the error message in #5770 doesn't provide a hint for user to read description of this flag.
Instead, I suggest adding a note to https://docs.victoriametrics.com/cluster-victoriametrics/#architecture-overview that vmselect still requires some disk space while being stateless.

Additionally, would be great to add to troubleshooting docs case related to all disk failures: no permissions, no disk space left, etc. We had a plenty of such issues before, so documenting this seems reasonable.

Reply:
> By default, the cache is stored at /tmp/searchResults

I could get the point, but this description might be incorrect. I'd like to ensure the user can understand that this flag is responsible for both **the rollup cache** and **the temporary query results**. How about the following explanation for this flag:

> Path to directory for cache files and temporary query results. By default, the cache won't be persisted, 
and temporary query results will be placed under /tmp/searchResults. If set, the cache will be persisted under cacheDataPath/rollupResult, and temporary query results will be placed under cacheDataPath/tmp/searchResults.
---
> adding a note to ... vmselect still requires some disk space while being stateless.

Agree. It often confuses people. I added following note:

> Note that `vmselect` still requires some disk space for temporary query results despite being stateless. Refer to the `-cacheDataPath` command-line flag for more details.